File: starter-bots/JavaBot/src/main/java/Models/GameState.java
package Models;

import java.util.*;

public class GameState {

    public World world;
    public List<GameObject> gameObjects;
    public List<GameObject> playerGameObjects;

    public GameState() {
        world = new World();
        gameObjects = new ArrayList<GameObject>();
        playerGameObjects = new ArrayList<GameObject>();
    }

    public GameState(World world , List<GameObject> gameObjects, List<GameObject> playerGameObjects) {
        this.world = world;
        this.gameObjects = gameObjects;
        this.playerGameObjects = playerGameObjects;
    }

    public World getWorld() {
        return world;
    }

    public void setWorld(World world) {
        this.world = world;
    }

    public List<GameObject> getGameObjects() {
        return gameObjects;
    }

    public void setGameObjects(List<GameObject> gameObjects) {
        this.gameObjects = gameObjects;
    }

    public List<GameObject> getPlayerGameObjects() {
        return playerGameObjects;
    }

    public void setPlayerGameObjects(List<GameObject> playerGameObjects) {
        this.playerGameObjects = playerGameObjects;
    }

}

File: starter-bots/NETCoreBot/NETCoreBot/Models/GameState.cs
using System.Collections.Generic;

namespace NETCoreBot.Models
{
    public class GameState
    {
        public World World { get; set; }
        public List<GameObject> GameObjects { get; set; } // [ size, x, y, type, id] DTO, non player objects
        public List<GameObject> PlayerGameObjects { get; set; } // [ size, x, y, speed, type, id] DTO
    }
}